### ISSUE TITLE
Add documentation about how to implement it in Expo

### ISSUE THREAD

**Issue by BrodaNoel** [DylanVann/react-native-fast-image#837]
It may be good to have some documentation about how to implement it in Expo

**Comment by ataravati** [comment]
@kaueDM How can it be the exact same thing? Linking doesn't exist in Expo, neither does `pod install`.

**Comment by kaueDM** [comment]
@ataravati Did you read the link I sent? If not, take a loot at this one:

https://docs.expo.dev/development/introduction/

Custom development clients will behave just like CLI apps. You just have to install the custom dev client lib:

`yarn add expo-dev-client`

and then install any native lib you want, like `fast image`

`yarn add react-native-fast-image`

That's it. To run locally, use `expo run:ios` or `expo run:android`. This `run` methods will create the iOS and Android folders, so you can do anything described on this repo instructions (e.g. `pod install`)

**Comment by ataravati** [comment]
@kaueDM Thanks! I'll give it a try.

**Comment by ansmlc** [comment]
> @ataravati Did you read the link I sent? If not, take a loot at this one:
> 
> https://docs.expo.dev/development/introduction/
> 
> Custom development clients will behave like CLI apps. You just have to install the custom dev client lib:
> 
> `yarn add expo-dev-client`
> 
> and then install any native lib you want, like `fast image`
> 
> `yarn add react-native-fast-image`
> 
> That's it. To run locally, use `expo run:ios` or `expo run:android`. This `run` methods will create the iOS and Android folders, so you can do anything described on this repo instructions (e.g. `pod install`)

@kaueDM Will this make it work in Expo Go?

**Comment by kaueDM** [comment]
No, you can't use native modules on expo go. That's why custom clients exists.

**Comment by ahmed-fayyaz-dev** [comment]
Can you still use EAS build service if you've setup iOS/android folders locally for expo-dev-client ?

**Comment by appsgenie** [comment]
@kaueDM your suggesting is good but that's more equivalent to an eject I think. Most of us want to stay with the managed expo flow even on EAS. So with custom dev clients in expo, I think we need a plugin to make it work no?

**Comment by kaueDM** [comment]
> @kaueDM your suggesting is good but that's more equivalent to an eject I think. Most of us want to stay with the managed expo flow even on EAS. So with custom dev clients in expo, I think we need a plugin to make it work no?

It still managed and you can build on EAS, I have being doing it for months. The only downside its that you can't use the Expo Go app to test, you need to build a custom dev client and run it in your emulator/device.

**Comment by BrodaNoel** [comment]
Take a look at 'expo-image'.

**Comment by appsgenie** [comment]
@kaueDM  yes that is ok with me as I can't use Expo Go anymore anyways but was just not sure if doing an install in my project without anything else will suffice.

@BrodaNoel I was looking at Discord and there are so many complaints about `expo-image` that I am a bit scared to install it yet :) plus was not able to get to the latest version because it said the correct version for my project is 1.0.1 but there are a few bug fixes in the latest version, so I think I would rather wait not sure..
